src/prisma/generator/templates/actions.py.jinja

```diff
@@ -232,8 +232,8 @@ class {{ model.name }}Actions(Generic[{{ ModelType }}]):
         )
         ```
         """
-        if self._client._active_provider == 'sqlite':
-            raise errors.UnsupportedDatabaseError('sqlite', 'create_many()')
+        if skip_duplicates and self._client._active_provider in ['mongodb', 'sqlserver', 'sqlite']:
```

Comment:
nit: can you share the Prisma docs that say this is unsupported? might be a good idea to add that link in a comment here as well? (applies to `client.py.jinja` as well)

Reply:
separately, this is a tiny nit but it'd be more efficient for this to be a set
```suggestion
        if skip_duplicates and self._client._active_provider in {'mongodb', 'sqlserver', 'sqlite'}:
```
it might also make sense to extract this to `_constants.py` as well? I don't feel strongly